src/GregClient/NuGet/CreateNugetPackage.ps1

```diff
@@ -0,0 +1,23 @@
+﻿param(
```

Comment:
This is a new powershell script that is added to generate the NuGet package. We need to use a powershell script because it's the only scripting component that enables us to the get the dll version to feed into the command to `pack` the NuGet package.